node/impls.go

```diff
@@ -135,6 +137,9 @@ type agreementLedger struct {
 
 	ff rpcs.FetcherFactory
 	n  network.GossipNode
+
+	// concurrent fetchByDigest kludge (until we remove fetchByDigest entirely)
+	ongoingFetchByDigest uint32 // 0 if no, 1 if yes
```

Comment:
place that as the first/second member of this struct to ensure it's 64-bit aligned.
It's needed for supporting RasPI.

Reply:
https://golang.org/pkg/sync/atomic/#pkg-note-BUG

Seems that this only applies to 64 bit words? not uint32